docs/organizations/managing-security-and-risk.md

```diff
@@ -34,7 +34,10 @@ The Security and Risk Management item list displays a list of items, filtered by
 
 To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
 
-To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
+When viewing the item list:
+
+-   To update the filtering criteria, click the **Status** dropdown or the **Priority** dropdown above the list.
+-   To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
```

Comment:
```suggestion
-   To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.<!-- TODO Depending on which feature is released first, this may need to change to take into account items coming from other integrations such as Jira -->
```